src/middlewared/middlewared/plugins/unscheduled_reboot_alert.py

```diff
@@ -37,10 +38,22 @@ class UnscheduledRebootAlertService(Service):
         if not await middleware.call('notifier.is_freenas') and await middleware.call('notifier.failover_licensed'):
             if os.path.exists(WATCHDOG_ALERT_FILE):
                 watchdog_file = True
-                watchdog_time = os.path.getmtime(WATCHDOG_ALERT_FILE)
+                try:
+                    fd = os.open(WATCHDOG_ALERT_FILE, os.O_RDONLY)
+                    b = os.read(fd, 4)
+                    watchdog_time = struct.unpack("@i", b)[0]
+                    os.close(fd)
+                except EnvironmentError:
+                    pass
```

Comment:
If some exception occurs watchdog_time will be undefined and cause exception later on.